<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaubliControlDlgUI</class>
 <widget class="QWidget" name="StaubliControlDlgUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Staubli - Barrett Control</string>
  </property>
  <widget class="QGroupBox" name="staubliStatusGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Staubli Status</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>url/un/pwd</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="staubliAddress">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>17</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>http://192.168.1.254:5653/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectToStaubliButton">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>15</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>16</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>default</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>16</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="getStaubliStatusButton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>100</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="staubliPoseLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>pose:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="staubliJointsLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>joints:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Staubli: x,y,z,rx,ry,rz</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="staubliControlGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Staubli Control</string>
   </property>
   <widget class="QLineEdit" name="staubliDestination">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="staubliGoButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>goto:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sendCommandToStaubli">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>send cmd to robot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Barrett: x,y,z,qw,qx,qy,qz</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Barrett Hand Status</string>
   </property>
   <widget class="QPushButton" name="initializeBarrettButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>82</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize hand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getBarrettStatusButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>82</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="barrettJointsLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>joints:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="barrettPoseLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>pose:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getBarrettButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>82</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Get hand</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Barrett Hand Control</string>
   </property>
   <widget class="QPushButton" name="barrettGoButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>47</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="barrettDestination">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>goto:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sendCommandToBarrett">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>56</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>send cmd to robot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>42</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dof0,dof1,dof2,dof3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="barrettCloseButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>47</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="barrettOpenButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>47</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getLocButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>get loc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
